src/Storefront/Resources/views/page/product-detail/index.html.twig

```diff
@@ -29,7 +29,7 @@
                                                 'mediaItems': mediaItems,
                                                 'zoom': true,
                                                 'zoomModal': true,
-                                                'displayMode': 'cover',
+                                                'displayMode': 'contain',
```

Comment:
Why did you change the display mode here?

Reply:
Yes please revert this change back to cover ;)

You can change the display mode in the settings of the gallery cms block:
![image](https://user-images.githubusercontent.com/15930605/67925198-013cf280-fbb3-11e9-8eaf-2c71f1315fa7.png)

Reply:
Updated the commit and reverted back.